cypress/integration/Crosscutting/HttpStatusCodeTest/index.js

```diff
@@ -0,0 +1,11 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('correct status code {string} is received for the {string}', (code, path) => {
+    cy.request({
+        url: path,
+        followRedirect: false, 
+      }).then((resp) => {
+        expect(resp.status).toString().match(code);      })
```

Comment:
this should be equal
` expect(resp.status).to.eq(code);`

Reply:
done